microshift.spec

```diff
@@ -0,0 +1,70 @@
+%{!?github_tag: %global github_tag 4.7.0-0.microshift-2021-08-31-224727}
+%{!?version: %global version 4.7.0}
+%{!?release: %global release 2021_08_31_224727}
+
+# golang specifics
+%global golang_version 1.15
+#debuginfo not supported with Go
+%global debug_package %{nil}
+# modifying the Go binaries breaks the DWARF debugging
+%global __os_install_post %{_rpmconfigdir}/brp-compress
+
+Name: microshift
+Version: %{version}
+Release: %{release}%{dist}
```

Comment:
where is `dist` defined?

Reply:
It comes from rpmbuild, normally the distro where you are building, or, a mock envionment created by the builder that you request for a specific distro.

I passed the output SRPM from this, and https://copr.fedorainfracloud.org/coprs/mangelajo/microshift/build/2829853/


I was thinking that we could automate that for every tag, and then have one for every new git commit in the form of "devel" version and "%{timestamp}.git%{commit-sha}" release.

Reply:
i see, thanks for the education.